src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -1,7 +1,7 @@
 ﻿---
 ################################
 # Assembly references report
-# Report date/time: 2020-10-27T14:54:08.8664584Z
+# Report date/time: 2020-10-29T09:42:23.0046223Z
```

Comment:
It doesn't look like anything has changed in the references, which is what I'd expect. However, in that case I wouldn't have expected the AsmRef targets to have updated the baseline in this case.

Committing these unnecessary changes is fine; I'm just confused why the tooling produced them. Did you make reference changes locally then revert them?

Reply:
Hm, I can't recall. I'll try to notice it if it happens again.